test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
> No, they are not.

They are queued in a binary heap within libuv. I am **very** certain that I am correct.

> I guess I'll also add/clarify that even If it seems like I'm being ridiculous and this will never ever cause problems itself, arbitrary-length timeouts in tests are a bit of an anti-pattern that have bitten us over and over.

I am well aware. (... do you really think that I am/was not?)

> arbitrary-length

That is why I suggested what I did in my last comment. It may be valuable for you to try to understand how timers work given you appear to feel so strongly about them...?